### PR TITLE
Applyconfig: Allow passing multiple config dirs

### PR DESCRIPTION
**Pull request by alvaroaleman**
Needed so we do not have to duplicate everything for app.ci.

/assign @petr-muller @hongkailiu

### CODE REVIEW

File: cmd/applyconfig/applyconfig.go [hunk 1]
```diff
@@ -302,7 +303,7 @@ func init() {
 
 func main() {
 	o := gatherOptions()
-	var adminErr, standardErr error
+	var adminErrs, standardErrs []error
```
Comment (bbguimaraes) [inline cmd/applyconfig/applyconfig.go]: These don't seem to be used directly, they could be replaced by a single `bool`.

File: cmd/applyconfig/applyconfig.go [hunk 2]
```diff
@@ -316,9 +317,11 @@ func main() {
 			return apply(o.kubeConfig, o.context, path, o.user.val, !o.confirm)
 		}
 
-		adminErr = applyConfig(o.directory, "admin", f)
-		if adminErr != nil {
-			logrus.Errorf("There were failures while applying admin config")
+		for _, dir := range o.directories.Strings() {
+			if err := applyConfig(dir, "admin", f); err != nil {
+				adminErrs = append(adminErrs, err)
+				logrus.WithError(err).Error("There were failures while applying admin config")
```
Comment (bbguimaraes) [inline cmd/applyconfig/applyconfig.go]: Do the errors returned by `applyConfig` have enough context to determine which file/directory? Would including `dir` in the log be helpful?

File: cmd/applyconfig/applyconfig.go (hunk 1 above)
Comment (alvaroaleman) [reply cmd/applyconfig/applyconfig.go]: Done

File: cmd/applyconfig/applyconfig.go (hunk 2 above)
Comment (alvaroaleman) [reply cmd/applyconfig/applyconfig.go]: Yeah, when we walk the dir we log each folder and we also log the `oc` command, so I think that should be enough

**Pull request merged**